draft-ietf-quic-transport.md

```diff
@@ -4786,17 +4838,28 @@ active_connection_id_limit (0x0e):
   When a zero-length connection ID is being used, the active_connection_id_limit
   parameter MUST NOT be sent.
 
+initial_connection_id (0x0f):
+
+: The value that the endpoint included in the Source Connection ID field of the
+  first Initial packet it sends during the handshake; see {{cid-auth}}.
```

Comment:
nit: I would s/packet it sends during the handshake/packet it sent on this connection/

Reply:
Everywhere else, "initial_connection_id" is defined as _The value that the endpoint included in the  **Destination** Connection ID field of the first Initial packet it sends during the handshake._
I think that the simplest solution is to s/Source/Destination/

Reply:
I don't think that's right @huitema. `initial_connection_id` refers to the Source Connection ID because that's the one that needs protection. The Destination Connection ID is covered by `original_connection_id `.

Reply:
@DavidSchinazi If you want to refer to the source CID set by the client in its first Initial packet, you should really use a different name than `initial_connection_id`, because this is widely used to refer to the initial destination CID. Maybe you should call that `initial_source_connection_id`. But using the same name for 2 concepts seems bad.

Reply:
I didn't pick the name, but I agree that `initial_source_connection_id` might be clearer than `initial_connection_id`.